File: Cpp/inheritance/Manusia.cpp
using namespace std;

class Manusia {
    private:
        string nama;
        string alamat;
        string no_ktp;
        string no_telp;

    public:

    Manusia(){
    }

    void setNama(string nama){
        this->nama = nama;
    }

    string getNama(){
        return nama;
    }

    void setAlamat(string alamat){
        this->alamat = alamat;
    }

    string getAlamat(){
        return alamat;
    }

    void setNoKtp(string no_ktp){
        this->no_ktp = no_ktp;
    }

    string getNoKtp(){
        return no_ktp;
    }

    void setNoTelp(string no_telp){
        this->no_telp = no_telp;
    }

    string getNoTelp(){
        return no_telp;
    }

    ~Manusia(){
    }
};

    

File: java/inheritance/Manusia.java
class Manusia{
    private String nama;
    private String alamat;
    private String no_ktp;
    private String no_telp;

    Manusia(){}

    public void setNama(String nama){
        this.nama = nama;
    }

    public String getNama(){
        return nama;
    }

    public void setAlamat(String alamat){
        this.alamat = alamat;
    }

    public String getAlamat(){
        return alamat;
    }

    public void setNoKtp(String no_ktp){
        this.no_ktp = no_ktp;
    }

    public String getNoKtp(){
        return no_ktp;
    }

    public void setNoTelp(String no_telp){
        this.no_telp = no_telp;
    }

    public String getNoTelp(){
        return no_telp;
    }
}